open-machine-learning-jupyter-book/assignments/data-science/matplotlib-applied.ipynb

```diff
@@ -33,8 +33,7 @@
     "* Scatter Plots, subplots\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    def clean_fare(df, ticket_price):
</pre><p>Please add some documentation to the clean fare method. Similar to <code>label_encode</code>.</p><p> </p><p>We can actually make this function more general like this and test against it, <code>def impute_with_mean(df, column_name, imputed_column_name):</code></p><p> </p><p>Then you can call it for price as this,  <code>impute_with_mean(df, 'price', 'price_add_average')</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/185/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>